backend/src/main/java/com/woowacourse/gongseek/question/application/QuestionService.java

```diff
@@ -0,0 +1,20 @@
+package com.woowacourse.gongseek.question.application;
+
+import com.woowacourse.gongseek.question.domain.Question;
+import com.woowacourse.gongseek.question.domain.repository.QuestionRepository;
+import com.woowacourse.gongseek.question.presentation.dto.QuestionRequest;
+import com.woowacourse.gongseek.question.presentation.dto.QuestionResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
```

Comment:
```suggestion
@RequiredArgsConstructor
@Service
```
이렇게 Lombok관련 어노테이션은 위쪽으로 올리는건 어떤가요?